packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -1041,10 +1041,18 @@ class ProjectFileInvalidator {
   static const String _pubCachePathWindows = 'Pub/Cache';
 
   static List<Uri> findInvalidated({
-    @required DateTime lastCompiled,
+    @required DateTime /*?*/ lastCompiled,
```

Comment:
nullable or not, I'd prefer we keep the comment based syntax out of the tool and wait for the real nnbd to be ready.

```suggestion
    @required DateTime lastCompiled,
```